backend/scripts/rollback.sh

```
@@ -0,0 +1,99 @@
+#!/bin/bash
+# This script performs a rollback to the previous working version with MySQL database restore
+
+# Load environment variables
+if [ -f ~/fin-invoice-ocr-team6/shared/.env ]; then
+  echo "Loading environment variables from .env file"
+  source ~/fin-invoice-ocr-team6/shared/.env
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)